.circleci/config.yml

```diff
@@ -1,15 +1,23 @@
 version: 2
 workflows:
   version: 2
-  test:
+  # test:
```

Comment:
Why are these all commented out?
I assume just as WIP while testing the other script, right?

Reply:
Yes, in order to test the new job. You can revert or remove the commit "Dactivate test workflow" later